sections/header.liquid

```diff
@@ -468,50 +470,52 @@
           </ul>
         </nav>
       {%- else -%}
```

Comment:
Here instead of having another `if` nested, you can change the `else` to an `elsif`
```suggestion
      {%- elsif section.settings.menu_type_desktop != 'drawer' -%}
```